modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase

```diff
@@ -1590,6 +1590,18 @@ definition {
 
 				ObjectPortlet.viewEntry(entry = "userfn${userNum}");
 			}
+
+			task ("Excluing Relationship entries") {
+				ObjectAdmin.openObjectAdmin();
+
+				ObjectPortlet.selectCustomObject(label = "Custom Object 156668");
+
+				ObjectAdmin.gotoRelationshipsTab();
+
+				ObjectAdmin.deleteRelationshipViaUI(
+					relationshipLabel = "Relationship",
+					relationshipName = "relationship");
+			}
```

Comment:
Hey @JerryAdrianoo,

Delete this task and lines, it's not necessary, because it's no goal of this testcase. Thanks!